sst/SimEngMemInterface.cc

```diff
@@ -168,6 +169,21 @@ void SimEngMemInterface::requestRead(const MemoryAccessTarget& target,
   AggregateReadRequest* aggrReq = new AggregateReadRequest(target, requestId);
   std::vector<StandardMem::Request*> requests =
       makeSSTRequests<AggregateReadRequest>(aggrReq, addrStart, addrEnd, size);
+  uint64_t id = requestId;
+#ifdef SIMENG_ENABLE_SST_TESTS
+  testReqCounter_++;
```

Comment:
Should we not just be leveraging the standard requestID source (i.e. the sequence ID of the instruction which generates this request)?

Reply:
Given we've restricted the archetype usage to only OoO

Reply:
Answered [here](https://github.com/UoB-HPC/SimEng/pull/250#discussion_r1007926325). Will be changing it back.